src/main/java/dinkplugin/notifiers/DiaryNotifier.java

```diff
@@ -72,8 +79,9 @@ public void onVarbitChanged(VarbitChanged event) {
         } else if (value > previous) {
             diaryCompletionById.put(id, value);
 
-            if (value < 2 && (id == 3578 || id == 3599 || id == 3611)) {
-                // Karamja special case: 0 = not started, 1 = started, 2 = completed tasks
+            if (!isComplete(id, value)) {
+                // Karamja special case
+                log.info("Skipping {} {} diary start (not a completion)", diary.getRight(), diary.getLeft());
```

Comment:
Would be nice to include the value here too

```suggestion
                log.info("Skipping {} {} diary start (not a completion with value {})", diary.getRight(), diary.getLeft(), value);
```

(might need formatting)